src/common/services/Database/subscribers/SensorSubscriber.ts

```diff
@@ -0,0 +1,45 @@
+/* istanbul ignore file */
+
+/* eslint-disable import/no-cycle */
+import {
+  EventSubscriber,
+  EntitySubscriberInterface,
+  InsertEvent,
+  UpdateEvent,
+  AfterInsert,
+  AfterUpdate,
+} from 'typeorm/browser';
+
+import { Sensor } from '../entities';
+
+import { SyncService } from '../../SyncService';
+
+@EventSubscriber()
+class SensorSubscriber implements EntitySubscriberInterface {
```

Comment:
As per my comment in `Database` (and won't say it more) I think injecting in the `SyncService` would be nice. This will mean we can more easily write tests. 

Few ways to do this. Couple I can think of is to `push` an instance into the `subscribers` array. Another being a closure

```
export const SensorSubscriber = (things) => {
    class Subscriber { 
         this.thing;
         constructor(){
              this.thing = thing
         }
          ......
    }
}
```

and when opening the connection `createConnection({...config, subscribers: SensorSubscriber(injectedThing), .. })`